include/deal.II/base/mpi.h

```diff
@@ -267,19 +267,6 @@ namespace Utilities
     class MPI_InitFinalize
     {
     public:
-
-      /**
-       * Constructor. Takes the arguments from the command line (in case of
-       * MPI, the number of processes is specified there), and sets up a
-       * respective communicator by calling <tt>MPI_Init()</tt>. This
-       * constructor can only be called once in a program, since MPI cannot be
-       * initialized twice.
-       *
-       * This constructor sets max_num_threads to 1 (see other constructor).
-       */
-      MPI_InitFinalize (int    &argc,
-                        char ** &argv) /*DEAL_II_DEPRECATED*/;
-
```

Comment:
Ah, bummer. I totally didn't see this. This change alone probably cost me 2 hours to get through with all of the consequent changes.

We can reintroduce the old behavior by giving the `max_num_threads` argument of the other constructor a default value of one. What do others think?